compiler/AST/symbol.cpp

```diff
@@ -778,12 +779,14 @@ bool ArgSymbol::isConstant() const {
   return retval;
 }
 
+// keep in sync with isConstValWillNotChange() in visibleFunctions.cpp
```

Comment:
Why doesn't that function call this method for ArgSymbols in addition to VarSymbols?